packages/imagelib/imagelib.20160413/descr

```diff
@@ -0,0 +1,17 @@
+The imagelib library implements image formats such as PNG or PPM
+
+The imagelib library implements image formats such as PNG or PPM  in
+OCaml, relying on only one external dependency: camlzip. However, we
+plan to reimplement zlib in OCaml at some point.
```

Comment:
Have you looked at https://github.com/oklm-wsh/Decompress ? It's a pure OCaml implementation of zlib